vault/store.go

```diff
@@ -74,8 +75,18 @@ func LoadStore(fullPath string) (*WalletStore, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	js, err := simplejson.NewJson(fileData)
+	value, ok := js.CheckGet("PrivateKeyEncrypted")
+	if ok {
+		privateKey := value.MustString()
+		js.Set("SeedEncrypted", privateKey[:64])
```

Comment:
What about using ed25519.SeedSize * 2? But no need to change if this will be removed later.

Reply:
Yes, it will be remove later.